### PR TITLE
feat: wire up some components

### PR DESCRIPTION
**Pull request by chungwwei**

### CODE REVIEW

File: src/component/ComposedTwoYAxisChart.tsx [hunk 1]
```diff
@@ -7,32 +7,29 @@ import {
   Legend,
   ComposedChart,
 } from "recharts";
+import { StreamDatapointModel } from "../models/stream_datepoint";
 
-function getRandomArbitrary(min: number, max: number) {
-  return Math.ceil(Math.random() * (max - min) + min);
+function formatData(data: StreamDatapointModel[]) {
+  const formatted = data.map((d, index) => {
```
Comment (juaoose) [inline src/component/ComposedTwoYAxisChart.tsx:13]: `index` is not used

File: src/component/Map.tsx [hunk 2]
```diff
@@ -70,25 +32,38 @@ export function Map() {
...
-      )}
+      {lowBitrateEvents.map(({ x, y, bitrate, timestamp }, i) => {
+        return (
+          <CircleMarker
+            center={[x, y]}
+            pathOptions={{ color: "red" }}
+            radius={10}
+          >
+            <Popup>
+              x: {x} <br />
```
Comment (juaoose) [inline src/component/Map.tsx:43]: I guess it could be nicer as latitude and longitude, if you agree.

File: src/screens/Analytics.tsx [hunk 3]
```diff
@@ -1,20 +1,220 @@
...
+      { x: 0, y: 0, audioBitrate: 0, timestamp: new Date() },
+    ],
+    healthCellMetaData: [
+      { x: 0, y: 0, bitrate: 0, timestamp: new Date() },
+      { x: 0, y: 0, audioBitrate: 0, timestamp: new Date() },
+    ],
+  });
+
+  useEffect(() => {
+    async function loadStreamData(streamId: string) {
```
Comment (juaoose) [inline src/screens/Analytics.tsx:52]: Would it be better to break this function into smaller ones so that its easier to follow?

**Review by juaoose — CHANGES_REQUESTED** [review]

**Comment by Leeoku** [comment]
did a general review and wrote what stood out right away. I'll do more indepth to learn/ask questions later

File: src/screens/Analytics.tsx [hunk 4]
```diff
@@ -28,44 +228,86 @@ export function Analytics() {
...
               <Tab label="Aggregate downstream" style={{ marginRight: 12 }} />
+              <Tab label="Individual modems" style={{ marginRight: 12 }} />
             </Tabs>
-            {value === 0 && <ComposedTwoYAxisChart />}
-            {value === 1 && <ComposedTwoAreasChart />}
-            {value === 2 && <ComposedTwoAreasChart />}
-            {value === 3 && <ComposedTwoAreasChart />}
-            {value === 4 && <StackAreasChart />}
-            {value === 5 && <StackAreasChart />}
+            {value === 0 && <ComposedTwoYAxisChart data={streamData} />}
```
Comment (Leeoku) [inline src/screens/Analytics.tsx:235]: a suggestion. Should we write a comment or change the value to match the tab name, make it readable for others?

**Review by Leeoku — APPROVED** [review]

**Review by juaoose — APPROVED** [review]

**Pull request merged**